notebooks/125-torchvision-zoo-to-openvino/125-lraspp-segmentation.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>logically, i think it is better to run pytorch then start convert and run openvino</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
reordered